contracts/bridge/BridgeTransferCommon.sol

```diff
@@ -113,7 +113,12 @@ contract BridgeTransfer is BridgeHandledRequests, BridgeFee, BridgeOperator {
         }
         for (i = sequentialHandleNonce; i <= maxHandledRequestedNonce && handledNoncesToBlockNums[i] > 0; i++) { }
         sequentialHandleNonce = i;
-        sequentialHandledRequestBlockNumber = handledNoncesToBlockNums[i-1];
+        // edge case: requestNonce zero is not yet handled.
+        if (sequentialHandleNonce == 0) {
+            sequentialHandledRequestBlockNumber = 1; // skip genesis block
+        } else {
+            sequentialHandledRequestBlockNumber = handledNoncesToBlockNums[i-1];
```

Comment:
` sequentialHandledRequestBlockNumber = handledNoncesToBlockNums[i-1];` is important. So why don't we write only this? :)
If we want make initial sequentialHandledRequestBlockNumber to 1, we can set in declare code. :)

Reply:
Fixed.